Yafc.Parser/Data/FactorioDataDeserializer_Entity.cs

```diff
@@ -406,6 +411,18 @@ private void DeserializeEntity(LuaTable table, ErrorCollector errorCollector) {
                 break;
             case "logistic-container":
                 goto case "container";
+            case "lightning-attractor":
+                if (table.Get("range_elongation", out int range) && table.Get("efficiency", out float efficiency) && efficiency > 0) {
+                    EntityAttractor attractor = GetObject<Entity, EntityAttractor>(table);
+                    attractor.energy = voidEntityEnergy;
+                    attractor.range = range;
+                    attractor.efficiency = efficiency;
+                    if (table.Get("energy_source", out LuaTable? energy) && energy.Get("drain", out string? drain)) {
+                        attractor.drain = ParseEnergy(drain) * 60;
```

Comment:
What is this 60? Games ticks per second?

I think it is good to define this somewhere (or at least add a comment explaining the magic number)